include/crm/common/output.h

```diff
@@ -111,6 +112,7 @@ typedef struct pcmk__message_entry_s {
  */
 pcmk__output_t *pcmk__mk_text_output(char **argv);
 pcmk__output_t *pcmk__mk_xml_output(char **argv);
+pcmk__output_t *pcmk__mk_html_output(char **argv);
```

Comment:
Let's remove the HTML stuff from this PR, and focus on just the lib/pengine XML format

Reply:
Done.